ruby/red-arrow/lib/arrow/slicer.rb

```diff
@@ -163,10 +163,30 @@ def reject(&block)
         RejectCondition.new(@column, block)
       end
 
+      def ends_with(substring, ignore_case: false)
```

Comment:
Other method including existing `match_substring` should have `?` suffix because this is Ruby API. :-)

```suggestion
      def end_with?(substring, ignore_case: false)
```